ArduPlane/tailsitter.cpp

```diff
@@ -84,6 +84,8 @@ void QuadPlane::tailsitter_output(void)
               integrator to the same throttle level
             */
             throttle = motors->get_throttle_hover() * 100;
+            float trim_throttle = plane.aparm.throttle_cruise;
+            throttle = MAX(throttle,trim_throttle) ;
```

Comment:
you might as well do this on one line.

```suggestion
            throttle = MAX(throttle,plane.aparm.throttle_cruise.get());
```

Reply:
you also have a rouge space before your semi colon

Reply:
will it cast properly? from double to float?

Reply:
I think it will be fine, probably worth double checking tho.

Reply:
builds and flys in SITL okay...will fix...thanks

Reply:
rogue, not rouge... took me a while to figure that one out :)

Reply:
all the right letters but not necessarily in the right order ;)